CMakeLists.txt

```diff
@@ -165,6 +165,7 @@ if(WITH_GUI)
   add_subdirectory(OrbitSsh)
   add_subdirectory(OrbitSshQt)
   add_subdirectory(OrbitQt)
+  add_subdirectory(webUI)
```

Comment:
Is there a reason to not start with a capital letter?

Reply:
Good point. I didn't even think about. I will change it. Thanks!

Reply:
Thanks.